api-server/src/main.ts

```diff
@@ -34,6 +31,10 @@ async function bootstrap() {
     await app.listen(PORT);
 
     console.info('application listening on port ', PORT);
+
+    if (AUTH_USE_TEST_USERS) {
+        console.warn(AUTH_USE_TEST_USERS_WARNING)
+    }
```

Comment:
I wonder if we could throw an exception if `AUTH_USE_TEST_USERS` is used in combination with `ENV=production`?